### ISSUE TITLE
Can't access Discord atm

### ISSUE THREAD

**Issue by adrifcastr** [gideonbot/Gideon#123]
Weird way to contact, but I'm currently not able to log into my Discord acc, contacted Trust & Safety in the hopings of getting this resolved. I have seen the recent commit so I suppose everything is still there, I hope at least.

**Comment by TheAcornHunter** [comment]
You haven’t been able to access it the whole day then?

**Comment by adrifcastr** [comment]
Yes, all I can do is wait for Trust & Safety to respond and hopefully restore my access.

In the meantime: [Telegram](http://t.me/adrifcastr)
